databases/redis/Portfile

```diff
@@ -117,4 +108,44 @@ The BSD-licensed community fork is valkey; you can install a redis-compatible po
     port install valkey +redis_compat
 "
 
+# Default/latest subport: redis @8.2.2
+subport redis {
+    version             8.2.2
+    # Uses global master_sites; distname defaults to redis-${version} because name=redis here
+    checksums           rmd160  bafe217ceea41a3ed719d08b52482de0f126f813 \
+                        sha256  4e340e8e822a82114b6fb0f7ca581b749fa876e31e36e9fbcb75416bec9d0608 \
+                        size    3901854
+
+    # Avoid co-install with valkey which ships same executables
+    conflicts           valkey
```

Comment:
this is the same for both subports and should thus be defined above in the main Portfile